frontend/packages/kubevirt-plugin/src/constants/vm/network/network-interface-model.ts

```diff
@@ -27,10 +37,16 @@ export class NetworkInterfaceModel extends ObjectEnum<string> {
   static fromString = (model: string): NetworkInterfaceModel =>
     NetworkInterfaceModel.stringMapper[model];
 
+  isDeprecated = () => this.deprecated;
```

Comment:
can we rather call it `isSupported` since the value still can be used (but shouldn't in the UI)? This is a probably different semantic than floppy.

Reply:
ok :-)